pyproject.toml

```diff
@@ -23,7 +23,9 @@ tqdm = "^4.54.0"
 matplotlib = "^3.3.3"
 patsy = "^0.5.1"
 pyia = "^1.2"
-lightkurve = "^1.11.3"
+lightkurve = ">=2.0b3"
+corner = "^2.1.0"
```

Comment:
I would keep lightkurve 1.11.3 for now as the mandatory version until version 2.0 is released. We can add version 2.0 to the dev-dependencies.

Reply:
We're going to have to be compatible with new lightkurve, it'll be released soon. Let's cross this hurdle when we get back to this tool!